mypy.ini

```diff
@@ -0,0 +1,14 @@
+[mypy]
+exclude = src/pymovements/plot.py
+
+[mypy-matplotlib.*]
```

Comment:
I think we can get rid of these ignores.
See the comment for `tox.ini`

Reply:
unfortunately we can't get rid of scipy ignore -- scipy does not type...